utils/iio_info.c

```diff
@@ -33,12 +34,16 @@
 static struct iio_context *ctx;
 
 static const struct option options[] = {
-	{0, 0, 0, 0},
+	{ "read-debugfs", no_argument, NULL, 'd' },
+	{ "no-read-attr", no_argument, NULL, 'n' },
+	{ }
 };
 
 static const char *options_descriptions[] = {
 	("[-x <xml_file>]\n"
 		"\t\t\t\t[-u <uri>]"),
+	("Read and print the value of debugfs attributes."),
+	("Do not read sysfs attributes, only print the names."),
```

Comment:
I would rephrase these in order to:
 - use  'debug' instead of 'debugfs'
 - create more consistency between the 2 descriptions

I suggest:

`("Read and print the value of debug attributes."),`
`("Do not read the value of regular attributes."),`